Source: @pullmoll
Upstream: no
Reason: This piece of code is ill-conceived and cannot work. Remove it.

--- a/rtosc/src/dispatch.c	2019-03-10 17:16:45.000000000 +0100
+++ b/rtosc/src/dispatch.c	2020-03-29 09:02:42.916119722 +0200
@@ -289,8 +289,7 @@
             return false;
     } else if(type == 4) {
         //extract substring
-        const char *sub=NULL;
-        return strstr(a,sub);
+        return false;
     } else if(type == RTOSC_MATCH_OPTIONS || type == 6) {
         return false;
     } else if(type == RTOSC_MATCH_ENUMERATED) {
